wasm/jit/jit_amd64.go

```diff
@@ -766,7 +766,128 @@ func (c *amd64Compiler) compileSub(o *wazeroir.OperationSub) error {
 		return err
 	}
 
-	// x1 += x2.
+	// x1 -= x2.
+	prog := c.newProg()
+	prog.From.Type = obj.TYPE_REG
+	prog.From.Reg = x2.register
+	prog.To.Type = obj.TYPE_REG
+	prog.To.Reg = x1.register
+	prog.As = instruction
+	c.addInstruction(prog)
+
+	// We no longer need x2 register after ADD operation here,
+	// so we release it.
+	c.locationStack.releaseRegister(x2)
+	return nil
+}
+
+func (c *amd64Compiler) compileMul(o *wazeroir.OperationMul) (err error) {
+	switch o.Type {
```

Comment:
It might be helpful to comment here that those unfamiliar with x86 instructions can look at https://www.felixcloutier.com/x86/index.html for explanations, noting why these are prefixed 'A' :p

Reply:
https://github.com/tetratelabs/wazero/pull/111/commits/1a057d8f22077c92adff6857f5b7a3405224eea5